test/Microsoft.ML.OnnxTransformerTest/OnnxTransformTests.cs

```diff
@@ -145,7 +145,9 @@ public void TestSimpleCase()
             pipe.GetOutputSchema(SchemaShape.Create(invalidDataWrongVectorSize.Schema));
             try
             {
-                pipe.Fit(invalidDataWrongVectorSize);
+                var onnxTransformer = pipe.Fit(invalidDataWrongVectorSize);
+                (onnxTransformer as IDisposable)?.Dispose();
```

Comment:
Does this mean that now we'll ask our users to actually dispose the onnx transformer? Should we add docs about it?

Reply:
Also, what happens if their pipeline consists of several transformers, will they need to extract the onnxtransformer and call dispose only on it?

Reply:
Yes, It should always be disposed. The leftover file happens only when loading from disk. But the inference session always gets left behind.`
I have updated the docs now. Thanks for the reminder.

Reply:
How will a user know there is an ONNX model within the pipeline they load? 

The ImageFeaturizer transform uses an ONNX model behind the scenes. Also, AutoML also loads its pipelines from disk.

Reply:
Tensorflow, Onnxruntime, ImageClassification are also cases where it is necessary to dispose the transformer. You are right that it is hard to assess all cases, especially in the AutoML scenarios. The safe approach is to dispose all models.